src/tools/perf/lib/uct_tests.cc

```diff
@@ -380,11 +379,7 @@ class uct_perf_test_runner {
         uct_perf_test_prepare_iov_buffer();
 
         sn = std::numeric_limits<uint8_t>::max();
-        if (CMD == UCX_PERF_CMD_AM) {
-            m_last_recvd_sn = sn;
-        } else {
-            set_sn(recv_sn, m_perf.uct.recv_mem.mem_type, &sn);
-        }
+        set_sn(recv_sn, m_perf.uct.recv_mem.mem_type, &sn);
```

Comment:
not sure it will work because recv_sn mem type can be not m_perf.uct.recv_mem.mem_type with AM